api/v1alpha1/ocplogtest_types.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// OcpLogtestSpec defines the desired state of OcpLogtest
+type OcpLogtestSpec struct {
+	// Image defines the docker image used for the benchmark
+	Image ImageSpec `json:"image"`
+
+	// length of each line
+	LineLength int `json:"line_length,omitempty"`
```

Comment:
Can we please stick to camelCase? So far we were using it consistently and line_length breaking this standard.

Reply:
Yeah its my GoLand. it auto completes for me. I missed some.